server/src/test/java/org/elasticsearch/snapshots/AbortedRestoreIT.java

```diff
@@ -75,20 +75,16 @@ public void testAbortedRestoreAlsoAbortFileRestores() throws Exception {
         );
 
         String indexName = "tbl";
+
         assertBusy(() -> {
-            RecoveryRequest request = new RecoveryRequest(indexName);
-            request
-                .indicesOptions(IndicesOptions.LENIENT_EXPAND_OPEN)
-                .activeOnly(true);
-            final RecoveryResponse recoveries = client().admin().indices().recoveries(request).get(5, TimeUnit.SECONDS);
-            assertThat(recoveries.hasRecoveries()).isTrue();
-            final List<RecoveryState> shardRecoveries = recoveries.shardRecoveryStates().get(indexName);
-            assertThat(shardRecoveries).hasSize(1);
-            assertThat(future.isDone()).isFalse();
-
-            for (RecoveryState shardRecovery : shardRecoveries) {
-                assertThat(shardRecovery.getRecoverySource().getType()).isEqualTo(RecoverySource.Type.SNAPSHOT);
-                assertThat(shardRecovery.getStage()).isEqualTo(RecoveryState.Stage.INDEX);
+            ClusterService clusterService = cluster().getInstance(ClusterService.class);
+            IndexMetadata indexMetadata = clusterService.state().metadata().index(indexName);
+            if (indexMetadata != null) {
```

Comment:
shouldn't this be a `assertThat(indexMetadata).isNotNull()` to trip the assertBusy and retry?

Reply:
Oops, yes, it should.  Have updated, thanks!

Reply:
thx!